example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -72,28 +52,63 @@ using BDataType   = Types::BDataType;
 using AccDataType = Types::AccDataType;
 using CDataType   = Types::CDataType;
 
-auto create_args(int argc, char* argv[])
+using Row = ck_tile::tensor_layout::gemm::RowMajor;
+using Col = ck_tile::tensor_layout::gemm::ColumnMajor;
+
+struct gemm_traits
```

Comment:
Please add documentation. This is our host API. Each user would start from here, so lets make our best to make a good first impression ;)